composer.json

```diff
@@ -51,8 +51,8 @@
         "symfony/routing": "^4.4",
         "symfony/security-acl": "^3.1",
         "symfony/security-bundle": "^4.4",
-        "symfony/security-core": "^4.4",
-        "symfony/security-csrf": "^4.4",
+        "symfony/security-core": "^4.4 || ^5.1",
```

Comment:
Looks like `symfony/security-core` is not `5.1`:

https://github.com/sonata-project/SonataAdminBundle/runs/1057543595#step:10:90

```bash
composer why-not symfony/security-core 5.1
symfony/security-bundle      v4.4.12  requires  symfony/security-core (^4.4)
```